editions/tw5.com/tiddlers/wikitext/Formatting in WikiText.tid

```diff
@@ -30,6 +30,31 @@ See [[Styles and Classes in WikiText]] for more formatting options
 
 The full list of KeyboardShortcuts can be found in the $:/ControlPanel -> ''Keyboard Shortcuts'' tab. 
 
-!! Formatting Semantics
+!! Semantic Information
 
-See [[Formatting in Wikitext Semantics]]
\ No newline at end of file
+|Wikitext | Resulting HTML Code  |h
+|Double single quotes are used for `''bold text''`|Double single quotes are used for `<strong>bold text</strong>` |
+|Double slashes are used for `//italic text//`|Double slashes are used for `<em>italic text</em>` |
+|Double underscores are used for `__underlined text__`|Double underscores are used for `<u>underlined text<u>` |
+|Double circumflex accents are used for `^^superscripted^^` text |Double circumflex accents are used for `<sup>superscripted</sup>` text |
+|Double commas are used for `,,subscripted,,` text |Double commas are used for `<sub>subscripted</sub>` text |
+|Double tilde signs are used for `~~strikethrough~~` text |Double tilde signs are used for `<strike>strikethrough</strike>` text |
+|Single backtick is used for ```code` `` |Single backtick is used for `<code>code</code>` |
```

Comment:
It would be better to say "Single backticks are used..." – super weird, I know, and I'm afraid I'm not familiar enough with grammar to cite the relevant rule!

Reply:
OK done